claudia.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
do we want to include this? Don't think we'll be deploying the app with claudia

Reply:
from talking to ian we dont. but currently this is my deployment path until someone shows me how to terraform it correctly.